oidc_provider/lib/endpoints/token.py

```diff
@@ -223,10 +227,18 @@ def create_code_response_dic(self):
         return dic
 
     def create_refresh_response_dic(self):
+        # See https://tools.ietf.org/html/rfc6749#section-6
+
+        scope_param = self.params['scope']
+        scope = (scope_param.split(' ') if scope_param else self.token.scope)
+        unauthorized_scopes = set(scope) - set(self.token.scope)
```

Comment:
nice catch from RFC :)
instead of subtracting maybe this approach?

`unauthorized_scopes = set(scope).issubset(set(self.token.scope))`